doit.go

```diff
@@ -40,26 +44,115 @@ func (c *LiveConfig) GetGodoClient() *godo.Client {
 	return godo.NewClient(oauthClient)
 }
 
+func sshConnect(user string, host string, method ssh.AuthMethod) (err error) {
```

Comment:
why do you name the return value? doesn't seem to be used in a defer block, which is the only reason to do that usually (unless the return values arent obvious just with their type)